profiles/models.py

```diff
@@ -2,8 +2,8 @@
 from datetime import datetime
 from typing import Any, Iterable, List, Optional
 
-from iso3166 import Country
 import pendulum
+from iso3166 import Country
```

Comment:
the linter (see build output) has opinions on import order and structure
```
profiles/models.py:6:1: I100 Import statements are in the wrong order. 'from iso3166 import Country' should be before 'import pendulum' and in a different group.
```
plus some other failing checks from static analysis

Reply:
I would disagree with the linter here. Is it ok if I update the linting rules?

Reply:
On second thought, i'm going to revert changes to imports for now and use a new format on newer projects.